web/rootfs/defaults/meet.conf

```diff
@@ -134,6 +135,20 @@ location ^~ /etherpad/ {
 }
 {{ end }}
 
+{{ if $WHITEBOARD_ENABLED }}
+# whiteboard (excalidraw-backend)
+location = /socket.io/ {
+    tcp_nodelay on;
+
+    proxy_http_version 1.1;
+    proxy_set_header Upgrade $http_upgrade;
+    proxy_set_header Connection "upgrade";
+    proxy_set_header Host $http_host;
+
+    proxy_pass http://whiteboard.meet.jitsi/socket.io/?$args;
```

Comment:
Note to self: In Etherpad we control this with ETHERPAD_URL_BASE, we shuld try to be consistent, but at the same time I'm not sure happy with how the Etherpad settings are done...